testing/dart/plugin_utilities_test.dart

```diff
@@ -43,4 +72,15 @@ void main() {
     final Function anon = (int a, int b) => a + b; // ignore: prefer_function_declarations_over_variables
     expect(PluginUtilities.getCallbackHandle(anon), isNull);
   });
+
+  test('PluginUtilities Callback Handles in Isolate', () async {
+    ReceivePort port = ReceivePort();
```

Comment:
I think we should skip this test like this:

```
if (!const bool.fromEnvironment('dart.vm.product')) {
  return;
}
```

I'd expec tthis test to pass for the host_release tests...

Reply:
That value is false even in host_release

Reply:
That's unfortunate. 

I don't think we should try to add more dart dfines for this. 

Perhaps the right option here would be to move this file to a different directory and update `run_tests.py` so that it only picks this file if doing the host_release build.

First, I would recommend verifying that this test does in fact pass on host_release.

Alternatively, you could write this as a shell_unittest and there you could check whether FLUTTER_RUNTIME_MODE is release or not in the C++ part of the test.

Reply:
(also, if it's easier to discuss this happy to talk more on discord or over a vc)